Dockerfile

```diff
@@ -78,6 +78,8 @@ RUN --mount=type=cache,target=/go/pkg/mod --mount=type=cache,target=/root/.cache
 
 FROM gcr.io/distroless/static as argoexec
 
+USER 8737
```

Comment:
Need some eyes on this

Reply:
Sure, let me know if you think this will take time, I can remove it from this PR & have it in a separate one. Don't want to make it release blocker for you! 😄

Reply:
Let's remove it for now

Reply:
Done thanks!!